src/Cake.XdtTransform.Tests/Cake.XdtTransform.Tests.csproj

```
@@ -1,15 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net48;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
 
     <IsPackable>false</IsPackable>
 
     <LangVersion>default</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Cake.Testing" Version="1.0.0" />
+    <PackageReference Include="Cake.Testing" Version="2.0.0" />
     <PackageReference Include="DotNet.Xdt" Version="2.2.1" />
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.1" />
```

Bump other dependencies too:
```suggestion
    <PackageReference Include="FluentAssertions" Version="6.2.0" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
```

---

Personally, I would rather bump the "different" dependencies in a separate PR, so the changes show up nicely in separate issues. 
However, I'm not against bumping those in the same PR, if @phillipsj is fine with that.

---

Sure bumping those in the same PR is fine with me.

---

@nils-a sure, no problem. I just used a suggestion for the convenience of expressing the change.

---

done.